### PR TITLE
Feat 387 skin docs

### PR DESCRIPTION
**Pull request by irina060981**
For the issue - https://github.com/alpheios-project/components/issues/387
I have created a very simple docs for creating a skin

### CODE REVIEW

File: development/skinning.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+Variable name | Default | Description
+------------ | ------------- | -------------
+`--alpheios-toolbar-bg-color` | #0E2233 | It is a dark color used for popup, panel headers and for the background of floating toolbar
+`--alpheios-border-color` | #757575 | It is used for toolbar border and lookup component border
+ |  | 
+`--alpheios-icon-color` | #FFF | It is used for toolbar, popup and panel icons (desktop and compact)
+`--alpheios-icon-color-hover` | #FFD27D | State color for icon color - hover
+`--alpheios-icon-color-active` | #FFC24F | State color for icon color - active, pressed
+`--alpheios-icon-color-disabled` | #757575 | State color for icon color - disabled
+`--alpheios-icon-color-disabled` | #757575 | State color for icon color - disabled
```
Comment (kirlat) [inline development/skinning.md]: This line is a duplicate of a previous one

File: development/skinning.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+ `--alpheios-lookup-bg` |  `--alpheios-text-bg-color` | Lookup panel panel background (in toolbar)
+ `--alpheios-lookup-input-border-color` | `--alpheios-border-color` | Lookup input border color
+ `--alpheios-lookup-button-color` | `--alpheios-btn-primary-font-color` | Lookup main button - font color
+ `--alpheios-lookup-button-bg` | `--alpheios-btn-primary-bg-color` | Lookup main button - background color
+ `--alpheios-lookup-button-border-color` | `--alpheios-btn-primary-border-color` | Lookup main button - border color
+ `--alpheios-lookup-button-color-hover` | `--alpheios-btn-primary-font-color-hover` | Lookup main button - font color - hover state
+ `--alpheios-lookup-button-bg-hover` | `--alpheios-btn-primary-bg-color-hover` | Lookup main button - background color - hover state
+ `--alpheios-lookup-button-border-color-hover` | `--alpheios-btn-primary-border-color-hover` | Lookup main button - border color - hover state
+ Inflections Browser |  | 
+ `--alpheios-inflect-browser-title-color` | `--alpheios-link-color-on-light` | Inflections Browser - main title - font color
```
Comment (kirlat) [inline development/skinning.md]: All following items starts with `--alpheios-inflect` prefix before the text describing the location. This one starts with `--alpheios-inflect-browsers`. Maybe it's better to use `--alpheios-inflect` for consistency?

File: development/skinning.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+ `--alpheios-inflect-button-control-color` | `--alpheios-btn-secondary-font-color` | Inflections Browser - expand/collapse button - font color
+ `--alpheios-inflect-button-control-bg` | `--alpheios-btn-secondary-bg-color` | Inflections Browser - expand/collapse button - background color
+ `--alpheios-inflect-button-control-border-color` | `--alpheios-btn-secondary-border-color`  | Inflections Browser - expand/collapse button - border color
+ `--alpheios-inflect-button-control-color-hover` | `--alpheios-btn-secondary-font-color-hover`  | Inflections Browser - expand/collapse button - font color - hover state
+ `--alpheios-inflect-button-control-bg-hover` | `--alpheios-btn-secondary-bg-color-hover`  | Inflections Browser - expand/collapse button - background color - hover state
+ `--alpheios-inflect-button-control-border-color-hover` | `--alpheios-btn-secondary-border-color-hover`  | Inflections Browser - expand/collapse button - border color - hover state
+ `--alpheios-inflect-match-cell-border-color` | `--alpheios-highlight-border-color` | Inflections Browser - border color for cells that has match
```
Comment (kirlat) [inline development/skinning.md]: Maybe it's better to indicate in a description that this is for cells that has a full feature match (if I remember correctly). Also, maybe it's better to rename it to something like `--alpheios-inflect-feat-match-cell-border-color` to signify that too?

File: development/skinning.md (hunk 2 above)
Comment (balmas) [reply development/skinning.md]: agree

File: development/skinning.md (hunk 3 above)
Comment (balmas) [reply development/skinning.md]: agree

Comment (irina060981) [reply development/skinning.md]: no, you are not right - it is not for full match, it is for morphologyMatch

`infl-cell--morph-match`: `cell.morphologyMatch`

I will add morphology to description

File: development/skinning.md (hunk 2 above)
Comment (irina060981) [reply development/skinning.md]: fixed

File: development/skinning.md (hunk 3 above)
Comment (kirlat) [reply development/skinning.md]: Great, thanks!

File: development/skinning.md (hunk 1 above)
Comment (irina060981) [reply development/skinning.md]: fixed

**Pull request merged**